makefiles/cflags.inc.mk

```diff
@@ -57,6 +57,26 @@ ifeq ($(LTO),1)
   LINKFLAGS += $(LTOFLAGS) -ffunction-sections -fdata-sections
 endif
 
+# Check if linker supports `-z noexecstack`. Handle BUILD_IN_DOCKER separately,
+# as this is run in the host environment rather than inside the container. We
+# just hardcode this in the BUILD_IN_DOCKER case for now.
+ifeq (1,$(BUILD_IN_DOCKER))
+  ifeq (msp430-elf,$(TARGET_ARCH))
+    LINKER_SUPPORTS_NOEXECSTACK := 0
```

Comment:
Hm somehow this does not work on CI

Reply:
This issue is that inside the docker `BUILD_IN_DOCKER` is `""`.